app/src/main/res/layout/topic_practice_header_view.xml

```diff
@@ -12,15 +12,14 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="32dp"
-    android:layout_marginTop="32dp"
-    android:layout_marginEnd="24dp"
-    android:paddingBottom="24dp">
+    android:layout_marginStart="@dimen/topic_practice_header_constraint_layout_margin_start"
+    android:layout_marginTop="@dimen/topic_practice_header_constraint_layout_margin_top"
+    android:layout_marginEnd="@dimen/topic_practice_header_constraint_layout_margin_end"
+    android:paddingBottom="@dimen/topic_practice_header_constraint_layout_padding_bottom">
 
     <TextView
       android:id="@+id/master_skills_text_view"
       style="@style/Heading2ViewStart"
-      android:layout_marginEnd="88dp"
```

Comment:
why you have removed this margin end?

Reply:
@yash10019coder not sure if we need it. There is nothing to the right of the textview.

Reply:
please don't remove it , it is required for RTL support

Reply:
i think i have already added it in dimens file.

Reply:
okay